alien/ArcaneInterface/modules/external_packages/src/alien/kernels/hypre/data_structure/HypreInternal.cc

```diff
@@ -29,7 +29,7 @@ VectorInternal::~VectorInternal()
 /*---------------------------------------------------------------------------*/
 
 bool
-MatrixInternal::init(const int ilower, const int iupper, const int jlower,
+MatrixInternal::init(const HYPRE_Int ilower, const HYPRE_Int iupper, const HYPRE_Int jlower,
```

Comment:
Why not changing also `jupper` and `lineSizes`? Or the other functions in the file?